src/main/java/frc/robot/Constants.java

```diff
@@ -18,11 +23,11 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-  public static final double ROBOT_MASS = (148 - 20.3) * 0.453592; // 32lbs * kg per pound
+  public static final double ROBOT_MASS_KG = (148 - 20.3) * 0.453592; // 32lbs * kg per pound
```

Comment:
```suggestion
    public static final Measure<Mass> ROBOT_MASS = Pounds.of(125); 
```

We don't know the weight but we can set it to the max for now.